xwiki-platform-core/xwiki-platform-index/xwiki-platform-index-api/src/main/java/org/xwiki/index/migration/R140300001XWIKI19571DataMigration.java

```diff
@@ -53,10 +53,13 @@
  *
  * @version $Id$
  * @since 14.3RC1
+ * @deprecated links indexing move to Solr, see org.xwiki.refactoring.internal.solr.* in
+ *             xwiki-platform-refactoring-default module
```

Comment:
Need to update the comment since it's not in xwiki-platform-refactoring-default anymore.